lib/generators/react_on_rails/bootstrap_generator.rb

```diff
@@ -0,0 +1,84 @@
+require "rails/generators"
+require File.expand_path("../generator_helper", __FILE__)
+include GeneratorHelper
+
+module ReactOnRails
+  module Generators
+    class BootstrapGenerator < Rails::Generators::Base
+      hide!
+      source_root(File.expand_path("../templates", __FILE__))
+
+      def append_to_assets_initializer
+        data = <<-DATA.strip_heredoc
+          # Add client/assets/ folders to asset pipeline's search path.
+          # If you do not want to move existing images and fonts from your Rails app
+          # you could also consider creating symlinks there that point to the original
+          # rails directories. In that case, you would not add these paths here.
+          Rails.application.config.assets.paths << Rails.root.join("client", "assets", "stylesheets")
+          Rails.application.config.assets.paths << Rails.root.join("client", "assets", "images")
```

Comment:
This seems problematic if we're using the sym links.

I see, generator is doing this, so we need to update the main readme.

this is good